asdf/tags/core/ndarray.py

```diff
@@ -240,6 +240,8 @@ def __init__(self, source, shape, dtype, offset, strides,
         self._offset = offset
         self._strides = strides
         self._order = order
+        if not asdffile.blocks._lazy_load:
+            self._make_array()
```

Comment:
Very minor, but I wonder if there should be an `@property` on the `AsdfFile` itself for this just to make the API a bit cleaner?

Reply:
Makes sense. I will add that and also fix the docstring.

Reply:
Thanks, the docstring needs to be fixed in both places.